gradle/test.gradle

```diff
@@ -249,6 +249,9 @@ subprojects {
         maxHeapSize '768m'
 //        jvmArgs = ['-XX:+HeapDumpOnOutOfMemoryError', '-ea',"-XX:+PrintGC", "-XX:+PrintGCDetails","-XX:+PrintGCTimeStamps"]
```

Comment:
Commented out code?

Reply:
that's not from this commit. That's from April 2016